internal/controllers/synthesis/lifecycle.go

```diff
@@ -68,8 +69,8 @@ func (c *podLifecycleController) Reconcile(ctx context.Context, req ctrl.Request
 
 	logger = logger.WithValues("compositionName", comp.Name,
 		"compositionNamespace", comp.Namespace,
-		"compositionGeneration", comp.Generation,
-		"synthesisID", comp.Status.GetCurrentSynthesisUUID())
```

Comment:
Replace with get latest uuid?

Reply:
I see, it's not relevant until we know that we're acting upon the in flight uuid.